website/project/model.py

```diff
@@ -736,9 +115,10 @@ def validate_title(value):
 
 
 def validate_user(value):
```

Comment:
If they're validators, sure.  There are a few others that look like utility methods that could just be  pushed onto their respective classes. Again, might be out of scope.